debian/control

```diff
@@ -32,10 +32,14 @@ Description: Provisioning backend for Homestead, the Cassandra powered HSS gatew
 
 Package: homer-cassandra
 Architecture: any
-Depends: clearwater-cassandra, clearwater-infrastructure
+Depends: clearwater-cassandra, clearwater-infrastructure, crest-prov
```

Comment:
I don't think they require it to function but the default behaviour is to not install recommended packages, right? If that's the case, we should tell chef/the build scripts to install it explicitly otherwise the user will have to do it manually.